### PR TITLE
Updated IssueCommentEvent

### PR DESCRIPTION
**Pull request by smit-modi**
Added EventType, Issue.Position, and Issue.BranchName.

### CODE REVIEW

**Comment by smit-modi** [comment]
@svanharmelen Looking for review

File: event_webhook_types.go [hunk 1]
```diff
@@ -246,6 +247,8 @@ type IssueCommentEvent struct {
 		HumanTimeEstimate   string        `json:"human_time_estimate"`
 		AssigneeIDs         []int         `json:"assignee_ids"`
 		AssigneeID          int           `json:"assignee_id"`
+		Position            int           `json:"position"`
+		BranchName          string        `json:"branch_name"`
```
Comment (svanharmelen) [inline event_webhook_types.go:251]: Can you order them according to the docs? So in between `AuthorID` and `Description`? Thanks!

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: event_webhook_types.go (hunk 1 above)
Comment (smit-modi) [reply event_webhook_types.go]: Fixed order.

**Review by svanharmelen — APPROVED** [review]
Thanks @smit-modi 👍🏻

**Pull request merged**